addons/mrp_subcontracting_dropshipping/views/purchase_order_views.xml

```diff
@@ -0,0 +1,16 @@
+<odoo>
+    <record id="view_purchase_order_inherit" model="ir.ui.view">
+        <field name="name">Purchase Order Inherit Dropship Subcontractor</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form"/>
+        <field name="arch" type="xml">
+            <field name="picking_type_id" position="after">
```

Comment:
even if the result is the same since the two fields are consecutive, I would write
`<field name="dest_address_id" position="before">`
given it is included in the attributes